### PR TITLE
Add cypress testing for home page layoutquote>

### LINKED ISSUE #14: Test Main Page
Main Page Should:
- [ ] Have a Header (Planet Party)
- [ ] Have a section/form with planet sort options
- [ ] Have planet names
- [ ] Have planet images with alt text

### PR DESCRIPTION
**Pull request by ConnorAndersonLarson**
# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?

- [ ] Fix
- [X] Feature
- [ ] Refactor
- [ ] Style
- [X] Testing

### Are there any known bugs?

- [X] No
- [ ] Yes (if so please disclose in Notes for Reviewer & Next Iterations)

### Summary of the changes
Adds Cypress to app along with basic tests for home screen layout.

### What ticket(s) do the changes resolve?
Close #14 
Please Link Issues to this PR as necessary.

### Notes for the Reviewer
Test fixture utilizes old API data structure, fixture/intercept means no api call is actually used.

### How to Test Changes?
Once in proper branch in one tab run `npm start` to begin local hosting of site and in a second tab run `npm run cypress` to open cypress and run tests.

### Next Iterations
We will need to switch intercept and possibly test date based on new API data.

### CODE REVIEW

File: cypress/integration/main-page.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    cy.get('.planet-card').should('have.length', 8)
+  })
+  it('Should have planets in order based on distance from sun to start', () => {
+    cy.get('.planet-card').eq(0).should('contain', 'Mercury')
+    cy.get('.planet-card').eq(1).should('contain', 'Venus')
+    cy.get('.planet-card').eq(4).should('contain', 'Jupiter')
+    cy.get('.planet-card').eq(5).should('contain', 'Saturn')
+    cy.get('.planet-card').eq(7).should('contain', 'Neptune')
+  })
+})
```
Comment (alexthompson207) [inline cypress/integration/main-page.js:32]: This is a great start, nice work!

**Review by alexthompson207 — COMMENTED** [review]
Great work with fixtures and the intercept, this is an awesome start!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,49 @@
 # PlanetParty
 Stretch Project in Mod 3 of Turing, learn/utilize Typescript
+
+# Getting Started with Create React App
```
Comment (knees4bees) [inline README.md:4]: We can change all of this `README` stuff later. Looks like it's just the template for Create React App.

File: cypress/fixtures/testPlanets.json [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+        "rotation_period": "655.7",
+        "length_of_day": "708.7",
+        "distance_from_sun": "0.384",
+        "orbital_period": "27.3",
+        "orbital_velocity": "1.0",
+        "mean_temperature": -20,
+        "number_of_moons": 0,
+        "created_at": "2017-11-12T22:46:36.599Z",
+        "updated_at": "2017-11-12T22:46:36.599Z"
+    },
```
Comment (knees4bees) [inline cypress/fixtures/testPlanets.json:69]: We can remove the Moon later.

File: cypress/fixtures/testPlanets.json [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+        "rotation_period": "-153.3",
+        "length_of_day": "153.3",
+        "distance_from_sun": "5906.4",
+        "orbital_period": "90.0",
+        "orbital_velocity": "4.7",
+        "mean_temperature": -225,
+        "number_of_moons": 5,
+        "created_at": "2017-11-12T22:46:36.617Z",
+        "updated_at": "2017-11-12T22:46:36.617Z"
+    }
```
Comment (knees4bees) [inline cypress/fixtures/testPlanets.json:171]: Same with Pluto -- can remove later. (And maybe add a separate area for dwarf planets?)

File: cypress/integration/main-page.js (hunk 1 above)
Comment (knees4bees) [inline cypress/integration/main-page.js:32]: Great job getting some of these tests set up!!

File: src/App/App.tsx [hunk 5]
```diff
@@ -24,7 +24,7 @@ class App extends React.Component<{}, AllData> {
     return (
       <div className="App">
         <header className="App-header">
-          <iframe src="https://giphy.com/embed/VI2UC13hwWin1MIfmi" className="giphy-embed"></iframe>
+          <iframe src="https://giphy.com/embed/VI2UC13hwWin1MIfmi" className="giphy-embed" data-alt="Gif of a cartoon Earth spinning; the Moon is spinning with it."></iframe>
```
Comment (knees4bees) [inline src/App/App.tsx]: per voice discussion, `data-alt` is telling TS that the description that follows is a string

**Review by knees4bees — APPROVED** [review]

**Comment by alexthompson207** [comment]
Resolved merge conflict - refactor header into `Header` component

**Pull request merged**